TUTORIAL.md

```diff
@@ -0,0 +1,396 @@
+```
+{
+  "name": "cesiumjs-webpack-tutorial",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "node_modules/.bin/webpack --config webpack.config.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "css-loader": "^6.2.0",
+    "html-webpack-plugin": "^5.3.2",
+    "style-loader": "^3.2.1",
+    "url-loader": "^4.1.1",
+    "webpack": "^5.50.0"
+  }
+}
+```
+
+Please note that details of this json file will vary based on your selections in step 3 of **Initialize an app with npm**.
+2. Run the command `npm run build`. Install CLI for webpack if necessary. webpack CLI can be installed using the command `npm install --save-dev webpack-cli`.
+3. Ensure that there are no errors and your output looks something like
+```
+$ npm run build
+
+> cesiumjs-webpack-tutorial@1.0.0 build
+> node_modules/.bin/webpack --config webpack.config.js
+
+asset app.js 1.22 KiB [emitted] (name: app)
+asset index.html 376 bytes [emitted]
+./src/index.js 28 bytes [built] [code generated]
+webpack 5.50.0 compiled successfully in 86 ms
+```
+Please note that the `app.js` bundle and `index.html` file should be added to the `dist` folder.
+
+## Run the development server
+1. We will be using a [webpack-dev-server](https://webpack.js.org/configuration/dev-server/) to serve a development build and see our application in action. Run `npm install --save-dev webpack-dev-server`.
+2. Add a `start` script to `package.json`. This script should run the development server. Be sure to set the config file via the `--config` flag and use the `--open` flag to open the application in a a browser upon execution of the command. `package.json` should look something like
+```
+{
+  "name": "cesiumjs-webpack-tutorial",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "node_modules/.bin/webpack --config webpack.config.js",
+    "start": "node_modules/.bin/webpack serve --config webpack.config.js --open"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "css-loader": "^6.2.0",
+    "html-webpack-plugin": "^5.3.2",
+    "style-loader": "^3.2.1",
+    "url-loader": "^4.1.1",
+    "webpack": "^5.50.0",
+    "webpack-cli": "^4.7.2",
+    "webpack-dev-server": "^3.11.2"
+  }
+}
+```
+3. Run `npm start` and open http://localhost:8080/ in a web browser.
+4. Ensure that you see Hello World!
+
+![South San Francisco](./screenshots/hw.png)
+
+## Add CesiumJS to a webpack app
+
+CesiumJS is a large and complex library. In additional to JavaScript modules, it also includes static assets such as CSS, image, and json files. It includes web worker files to perform intensive calculations in separate threads. Unlike traditional npm modules, CesiumJS does not define an entry point because of the diverse ways in which the library is used. We’ll need to configure some additional options to use it with webpack.
```

Comment:
> CesiumJS is a large and complex library

Not sure this should be the first line. Maybe introduce what Cesium does first, and then follow with this.